### ISSUE TITLE
LaTeX blocks not rendering

### ISSUE THREAD

**Issue by Lucretia** [shd101wyy/vscode-markdown-preview-enhanced#156]
No amount of ctrl-shift-enter's or shift-enter's do anything. I have texlive and pdf2svg packages installed, I#m getting nothing going to the assets/ directory. I'm on Gentoo.

**Comment by ketozhang** [comment]
It's not rendering on what?  The preview or an export?

**Comment by shd101wyy** [comment]
It would be great if you could send me a copy of your markdown file. Thanks.

**Comment by Lucretia** [comment]
I was testing the code from the website, the simple hello world and the code from #700

**Comment by Lucretia** [comment]
It'd be good to have a full list of dependencies, LaTeX tools and node packages, if any, in the readme so people can check they have everything required.

**Comment by shd101wyy** [comment]
I just tried it on my Ubuntu 18.04 and it works well. Below are what I installed

```bash
sudo apt install -y pdf2svg texlive texlive-latex-extra
```

I am not very sure about Gentoo though.

https://shd101wyy.github.io/markdown-preview-enhanced/#/extra?id=extra

**Comment by shd101wyy** [comment]
> You must know what parts of LaTeX you are using in this program???

Yes you are right. We basically call latex command to compile your code to pdf, then convert pdf to svg.


btw did you enable the `enableScriptExecution` option in extension settings?

**Comment by Lucretia** [comment]
No, I didn't see that mentioned in the docs.

**Comment by lamafab** [comment]
@Lucretia Did you ever manage to solve the LaTex rendering issue?

**Comment by Lucretia** [comment]
I haven't tried it.
